File: day one/day one python/Taskthree.py
sum = 0

average = 0;


for score in range(1,11):

	scores = int(input('Enter A Score: '))

	sum += score;

	average = sum / scores

print("The sum of The Scores Is: " + str(sum))

print("Average Of The Scores Is: " + str(average))

File: day two/java/TaskThree.java
import java.util.Scanner;

public class TaskThree{

	public static void main(String[] args){

	Scanner input = new Scanner(System.in);

	int[] numbers = new int[10];

	for(int counter = 0; counter < 10 ; counter++){

	System.out.print("Enter A Score: ");

	numbers[counter] = input.nextInt();


	}

	for(int count = 0; count < 10; count++){

	System.out.println( numbers[count] );

	}


	}







}